src/Notification.cpp

```diff
@@ -0,0 +1,41 @@
+#include "Notification.h"
+#include <coreinit/cache.h>
+#include <coreinit/thread.h>
+#include <coreinit/title.h>
+#include <notifications/notification_defines.h>
+#include <notifications/notifications.h>
+#include <thread>
+
+std::unique_ptr<std::thread> sShowHintThread;
+static bool sShutdownHintThread = false;
+static char * notification;
+
+void ShowNotification() {
```

Comment:
If you go with the parameter approach:
```suggestion
void ShowNotification(const char * notification) {
```
Otherwise if you want to keep it like this, this could be static.